src/Graphs/abstractgraph.jl

```diff
@@ -384,6 +386,31 @@ function mincut_partitions(graph::AbstractGraph, distmx=weights(graph))
   return parts[1], parts[2]
 end
 
+function insert_vertex!(graph::AbstractGraph, vertex)
+  z = add_vertex!(graph, vertex)
+  if !z
+    error("Duplicate vertices are not allowed")
+  else
+    return graph
+  end
```

Comment:
```suggestion
end
return graph
```
I've started to prefer this style, it makes it clearer what is being returned (instead of hiding the return inside a clause of the if-statement). Same for `delete_vertex!`.